test/functional/api/cas/cli.py

```diff
@@ -24,6 +24,14 @@ def script_try_add_cmd(cache_id: str, core_dev: str):
            f"--core-device {core_dev}"
 
 
+def script_purge_cache(cache_id: str):
+    return f"{casadm_bin} --script --purge-cache --cache-id {cache_id}"
+
+
+def script_purge_core(cache_id: str, core_id: str):
```

Comment:
Add *cmd* sufix to name like in other functions.
You may add `bool shortcut` as an optional parameter, which might be used in CLI tests.

Reply:
Script commands dont have shortcuts